x-pack/plugins/maps/common/get_geometry_counts.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Feature } from 'geojson';
+import { GEO_JSON_TYPE, VECTOR_SHAPE_TYPE } from './constants';
+
+export interface IVectorShapeTypeCounts {
```

Comment:
Nit: Can you change name from "IVectorShapeTypeCounts" to "VectorShapeTypeCounts". We have been using the "I" prefix to signify an interface that will have implemenations. This is just an object with some keys. To keep naming consistent with other types, this type should not have "I" prefix.